stacks/ecs.yml

```diff
@@ -141,7 +144,7 @@ Resources:
     Properties:
       ServiceToken: !GetAtt ECSAmiLookupFunction.Arn
       Region: !Ref AWS::Region
-      Rotation: 1 # Change this to trigger AMI updates
+      AmiName: !FindInMap [EnvironmentTypeMap, !Ref EnvironmentType, AmiName]
```

Comment:
I'm _hoping_ that this will do the same thing as rotation ... only call the Lambda when the `AmiName` value changes.

Reply:
My understanding is that it should work that way